src/downloadmanager.cpp

```diff
@@ -927,6 +927,31 @@ void DownloadManager::visitOnNexus(int index)
   }
 }
 
+void DownloadManager::openFile(int index)
+{
+  if ((index < 0) || (index >= m_ActiveDownloads.size())) {
+    reportError(tr("VisitNexus: invalid download index %1").arg(index));
+    return;
+  }
+  QString params = "/select,\"";
+  QDir path = QDir(m_OutputDirectory);
+  if (path.exists(getFileName(index))) {
+    params = params + QDir::toNativeSeparators(getFilePath(index)) + "\"";
+
+    ::ShellExecuteW(nullptr, nullptr, L"open", ToWString(params).c_str(), nullptr, SW_SHOWNORMAL);
+    return;
+  }
+  else if (path.exists(getFileName(index) + ".unfinished")) {

```

Comment:
we really don't people to be able to open an unfinished download

Reply:
Ah! Missed that. I didn't include the option in the non-`READY` states though, so I guess this `else if` would never get called… but if it'll never get called there's also no reason to have it, so: gone!